cmd/rpcdaemon/commands/eth_api.go

```diff
@@ -311,6 +326,10 @@ func NewEthAPI(
 	}
 }
 
+func (api *APIImpl) relayToHistoricalBackend(ctx context.Context, result interface{}, method string, args ...interface{}) error {
+	return api.historicalRPCService.CallContext(ctx, result, method, args...)
+}
```

Comment:
@pcw109550 Do you think we need this additional wrapping?

Reply:
Let me sort out the pros and cons of this abstraction.

Pros
1. Readability. Using functions makes you `name` the code. It becomes much easier to browse code without investigating the internals, what you read is what you get.
2. Maintenance. We only have to fix this function, not touching `CallContext` itself.

Cons
1. Performance. Wrapping adds additional call stack, slowing down the code.

I think advantages outweigh all the cons.